Src/DeUrgenta.Backpack.Api/Controllers/BackpackItemController.cs

```diff
@@ -39,7 +42,13 @@ public BackpackItemController(IMediator mediator)
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(ApplicationErrorResponseExample))]
         public async Task<ActionResult<IImmutableList<BackpackItemModel>>> GetBackpackItemsAsync([FromRoute] Guid backpackId)
         {
-            throw new NotImplementedException();
+            var query = new GetBackpackItems(backpackId);
```

Comment:
This method will be called only by authorized users but in order to find out which user called this method we extract user sub from claims.
To get user sub:
`var sub = User.Claims.FirstOrDefault(c => c.Type == "sub")?.Value;`

Please pass user sub to each command/query and use it to identify users in validators and handlers(if needed)